src/researchhub/settings.py

```diff
@@ -310,6 +310,13 @@ def silky_capture(request):
 
 PAGINATION_PAGE_SIZE = 10
 
+DEFAULT_RENDERER_CLASSES = ("rest_framework.renderers.JSONRenderer",)
+
+if DEBUG:
+    DEFAULT_RENDERER_CLASSES = DEFAULT_RENDERER_CLASSES + (
+        "rest_framework.renderers.BrowsableAPIRenderer",
+    )
+
```

Comment:
I would remove this as well, so the application behaves the same across any environment.

Reply:
I didn't want to disrupt Kobe's workflow if he uses it but I agree that all environments should be consistent. @yattias  thoughts?